### PR TITLE
Revert "GitHub Package Registry is now GitHub Packages"

### PR DESCRIPTION
**Pull request by Olivierdub38**
Reverts actions/starter-workflows#250

### CODE REVIEW

**Comment by Olivierdub38** [comment]
Olivierdub38:revert-250-ethomson/npmpublish

File: ci/properties/npm-publish.properties.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Node.js Package",
```
Comment (Olivierdub38) [inline ci/properties/npm-publish.properties.json]: ci/properties/npm-publish.properties.json[](url)

**Comment by abc52090241** [comment]
希望能解決問題

**Comment by ethomson** [comment]
Hello!  It looks like you may have opened this pull request accidentally, or without all the changes that you wanted to include.  As a result, I'm going to close this, but please do follow up if there are changes that you want to make here.

**Pull request closed without merging**